docs/content/latest/api/ycql/expr_fcall.md

```diff
@@ -60,6 +61,27 @@ function_call ::= function_name '(' [ arguments ... ] ')'
 | MIN | Returns the minimum value of column values |
 | MAX | Returns the maximum value of column values |
 
+
+## Partition_hash function
+
+YugabyteDB uses `partition_hash` function that returns an `uint16` hash of `partition_keys` to distribute rows internally. 
+The full hash partition value spans from `0 - 65535`. 
+We can use this function to query a subset of the data and get approximate results or do a parallel full table scan.
+
+### Querying a subset of the data
+Assuming we have a table:
+```sql
+create table t (h1 int, h2 int, r1 int, r2 int, v int, 
+                         primary key ((h1, h2), r1, r2));
+```
+We can use this function to query a subset of the data (in this case, 1/256 of the data):
```

Comment:
256 is a bad example (because 256 * 256 is 64K). So while example is correct, it isn't a good one.

It leads one to believe that if you want to query 1/16th of the rows the query would be:

partition_hash(h1, h2) < 16

A better example would be:

If you want to query 1/128th of the data, the query would be:

... partition_hash(h1, h2) < 512  

because 64K/128 = 512